Updated - index.html

NegiAkash890 opened the pull request:
This issue fixes some links as mentioned in the #2262 .

NegiAkash890 committed:
Updated - index.html

This fixes some  broken links in the Page .

chrisdavidmills reviewed (changes requested):
This is much better @NegiAkash890 ; thanks for your work so far. I just had two requests here:

1. There is a completely empty `<a>` element at the start of the paragraph; can you remove it?
2. Really, this paragraph should be an unordered list, with a separate link inside each list item. Can you update the markup as I've suggested.

NegiAkash890 committed:
Updated -  index.html

This removes the top empty anchor tag .
Removal of paragraph tag and addition of unordered list .

chrisdavidmills reviewed (changes requested):
Looking much better already!

One more thing — you can now get rid of the `<br/>` elements. They were present to create line breaks between the different links, but `<li>` are block-level elements so will create their own breaks.

NegiAkash890 committed:
Update - index.html

This removes line breaker  tags .

chrisdavidmills committed:
Tweak list spacing

chrisdavidmills reviewed (approved):
OK, this is good to go; thanks so much for your help @NegiAkash890 !
